### PR TITLE
Docs steps to make v0.11 docs live

### LINKED ISSUE #3704: TOC sidebar is out of date for v0.10 and main and missing for v0.11 release
## Description

When the versioned docs are generated, the sidebar for the docs are configured in `*-toc.yaml` files. Since we hadn't been working with versioned docs in the past, the v0.10 TOC contains some things that were not part of that release. The `main-toc.yaml` file contains different packages than what is currently in the repo.

Once `main-toc.yaml` is updated, that is likely going to match what the v0.11 release TOC should be, though that should be checked to make sure there aren't minor differences. But once reconciled, `main-toc.yaml` can probably be copied to be the TOC for the v0.11 release branch.

## Location

Documentation table of contents for each version.

## Additional Context

I think `main` is used for `edge`, but that should also be confirmed.

### PR DESCRIPTION
**Pull request by kcoriordan**
Signed-off-by: kcoriordan <koriordan@vmware.com>

## What this PR does / why we need it
Steps to make the v0.11 docs live: 

In `config.yaml`:
change `docs_latest `to `v0.11`
add `v0.11` to `docs_versions`
Copy `main-toc.yml `and rename copy as `v0.11-toc.yml`
Add new  row  in `toc-mapping.yml`: `v0.11: v0.11-toc
`
## Which issue(s) this PR fixes

Fixes: #3704

### CODE REVIEW

**Comment by kcoriordan** [comment]
@jonasrosland 
Can you check if this PR covers steps to make v0.11 docs live. thanks

**Comment by dvonthenen** [comment]
awesome! thank you!

**Review by davidvonthenen — APPROVED** [review]

File: docs/site/data/docs/v0.11-toc.yml [hunk 1]
```diff
@@ -0,0 +1,274 @@
+toc:
```
Comment (joshrosso) [inline docs/site/data/docs/v0.11-toc.yml:1]: @joshrosso TODO verification that this is consistent with `main`.

**Review by jonasrosland — APPROVED** [review]
LGTM, just need verification as Josh stated

File: docs/site/data/docs/v0.11-toc.yml (hunk 1 above)
Comment (dvonthenen) [reply docs/site/data/docs/v0.11-toc.yml:1]: Yup this is good to go

**Pull request merged**